package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.4",
+  "version": "2.79.5-hotfix-missing-payments-tab-alpha",
```

Comment:
Don't forget to make this a proper version number before merging into master

Reply:
Argh! Good call! That change will wipe out your approval too! One sec...

Reply:
It still isn't? It's still a pre-release version.

Reply:
Changed it to `2.79.5`.